dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops

```diff
@@ -229,3 +229,43 @@ LOGINPROPERTY_OVERRIDE_USERFILTER.useraction=If the userFilter attribute is pref
+		
+KRB5_LOGIN_FAILED_DEFAULT_CACHE=CWIML4509E: Kerberos login failed using Kerberos principal {0} and the default Kerberos credential cache (ccache).
+KRB5_LOGIN_FAILED_DEFAULT_CACHE.explanation=Either the specified Kerberos principal is invalid, or the default Kerberos credential cache (ccache) is invalid or expired.
+KRB5_LOGIN_FAILED_DEFAULT_CACHE.useraction=Ensure that a valid Kerberos principal is specified, and that the default Kerberos credential cache (ccache) is not expired.
+
+KRB5_LOGIN_FAILED_DEFAULT_KEYTAB=CWIML4510E: Kerberos login failed using Kerberos principal {0} and the default Kerberos keytab.
+KRB5_LOGIN_FAILED_DEFAULT_KEYTAB.explanation=Either the specified Kerberos principal is invalid, or the default Kerberos keytab is invalid.
+KRB5_LOGIN_FAILED_DEFAULT_KEYTAB.useraction=Ensure that a valid Kerberos principal is specified and that the default Kerberos keytab is valid.
+
+KRB5_TICKETCACHE_USED=CWIML4511E: The {0} LdapRegistry is configured with a Kerberos ticket cache (ccache) filename {1} and keytab filename {2}. The Kerberos credential cache (ccache) is used for Kerberos bind authentication to LDAP server.
+KRB5_TICKETCACHE_USED.explanation=In order to use the keytab file, the keytab filename must be specified and the Kerberos credential cache (ccache) filename must not be specified.
+KRB5_TICKETCACHE_USED.useraction=No action is required. To avoid this message, remove either the LdapRegistry ticketCache attribute or the Kerberos keytab attribute.
+
+INVALID_KRB5_PRINCIPAL=CWIML4512E: The {0} Kerberos principal name is incorrectly formatted, or the realm name is missing, or a default realm name cannot be found.
+INVALID_KRB5_PRINCIPAL.explanation=The Kerberos principal name cannot be null. The principal name must either include the realm name or a default realm name must be defined in Kerberos configuration file.
+INVALID_KRB5_PRINCIPAL.useraction=Correct the principal name or add a default realm name.
+
+CANNOT_READ_KRB5_FILE=CWIML4513E: The {0} LdapRegistry cannot read the Kerberos file at {1}.
+CANNOT_READ_KRB5_FILE.explanation=The Kerberos file cannot be opened. Either the file permissions are incorrect, or the file does not exist.
+CANNOT_READ_KRB5_FILE.useraction=Verify that the file location is correct and that the server has read file permissions.
+
+KRB5_PRINCIPAL_REQUIRED=CWIML4514E: The bind authentication mechanism on the {0} LdapRegistry is set to GSSAPI, but the required attribute, krb5Principal, is not configured.
+KRB5_PRINCIPAL_REQUIRED.explanation=The Kerberos principal name is required when GSSAPI (Kerberos) is enabled.
+KRB5_PRINCIPAL_REQUIRED.useraction=Add a krb5Principal attribute to the krbAuthentication elemenent on the LdapRegistry element.
+
+KRB5_FILE_NOT_FOUND=CWIML4515E: The {0} attribute from the {1} element is configured to a file that does not exist at: {2}
```

Comment:
The {0} attribute from the {1} element is configured to a file that does not exist at the following location: {2}